.github/dependabot.yml

```
@@ -15,6 +15,10 @@ updates:
       - dependency-name: "@automattic/*"
       - dependency-name: "@electron-forge/*"
       - dependency-name: "@wordpress/*"
+      - dependency-name: "@types/*"
+      - dependency-name: "eslint*"
+      - dependency-name: "eslint-*"
```

Do we need both patterns? I think `eslint*` also covers the `eslint-*` pattern.

```suggestion
      - dependency-name: "eslint*"
```

---

I think the idea was to catch `eslint` and packages like `eslint-plugin-import` but avoid catching `eslinternal`. I fixed it.